taichi/codegen/codegen_utils.h

```diff
@@ -8,4 +8,12 @@ inline bool codegen_vector_type(const CompileConfig &config) {
   return !config.real_matrix_scalarize;
 }
 
+inline std::string merge_printf_specifier(
+    std::optional<std::string> const &from_user,
+    std::string const &from_data_type,
+    Arch arch) {
+  // TODO: implement specifier merging
```

Comment:
Can you put an example in the  comment to explain what "specifier merging" be like?

Reply:
I added some comments~